academy/1-what-is-cosmos/index.md

```diff
@@ -4,16 +4,16 @@ parent:
   description: From the vision to the ecosystem
   number: 1
 order: 0
-title: Chapter Overview - The Interchain, Its Ecosystem, and ATOM
+title: Chapter Overview - The interchain, its ecosystem, and ATOM
```

Comment:
The title should follow title capitalization. I would go with `The Interchain, Its Ecosystem, and ATOM`.